pom.xml

```diff
@@ -113,6 +128,17 @@ THE SOFTWARE.
             <version>2.0.1</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
+            <version>1.6</version>
```

Comment:
Should be 1.4.1.
https://github.com/jenkinsci/jenkins/blob/jenkins-1.596.1/pom.xml#L99